neqo-transport/src/recovery/mod.rs

```diff
@@ -327,7 +327,7 @@ impl LossRecoverySpace {
             .sent_packets
             .iter_mut()
             // BTreeMap iterates in order of ascending PN
-            .take_while(|p| p.pn() < largest_acked.unwrap_or(PacketNumber::MAX))
+            .take_while(|p| p.pn() < largest_acked.unwrap_or(0))
```

Comment:
Given that `Option` implements `Ord` when the inner type implements `Ord`, this could as well revert to the previous implementation, namely:

``` diff
-            .take_while(|p| p.pn() < largest_acked.unwrap_or(0))
+            .take_while(|p| Some(p.pn()) < largest_acked)
```



https://github.com/mozilla/neqo/commit/5d54da7e22f187e4d60daad486830f9d25bb4d83#diff-69602dcab54959a67ff71f9fbdcd82eed8f15dee641f913ef12ffe8c59ad7f73

I am fine either way.